content/updates/productivity/2-Easier-collaboration-with-increased-import-and-export-capabilities.md

```diff
@@ -0,0 +1,49 @@
+# Easier collaboration with increased import and export capabilities
+
+## Import and export your reference entity records
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/import-export-records.html"
+
+Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in Excel or CSV, once enriched. So neat!
+
+You can find this new feature in the `Import` and `Export` menus, under the following job names: 
+- `Reference entity record import in XSLX`,
+- `Reference entity record import in CSV`,
+- `Reference entity record export in XLSX`,
+- `Reference entity record export in CSV`.
+
+![Record export profiles](../img/record-export-profiles.png)
+
+This new feature can also help you bulk update your records or easily duplicate them.
+
+## Export your products with their assets
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/exports.html#update-an-export-profile"
+
+It's now possible to export products and their associated assets from the product grid or your product export profiles. Collaboration around product data is more efficient and it will save you time whenever you need to share product information with your internal and external teams.
+
+To do so, you can use our new `With media` option when doing a quick export or using your favorite product export profile. Note, that by default, this option is not enabled.
+![Quick export products with their assets](../img/quick-export-products-with-assets.png)
+
+## Import assets with Excel or CSV files
+::: meta-data type="New" ee-only="true" link-to-doc="../articles/import-assets.html"
+
+To save you a tremendous amount of time, it is now possible to mass import assets via Excel or CSV files containing all your asset values. Indeed, in our `Import` menu, you can now discover a brand new import profile dedicated to this task. 
+
+![Asset import profile](../img/asset-import-profile.png)
+
+::: info
```

Comment:
I'd remove the info and maybe add it in the text above. Also, it could be confusing, how do we define "media file": what it is, and what it is not? Not sure what it covers. Or should we just say you can import assets used as main media for your products..

Reply:
Fixed with Laure